src/scenes/Jarjestajat/Jarjestaja/modules/constants.js

```diff
@@ -384,12 +384,12 @@ export const LUPA_TEKSTIT = {
                 SV: 'Luonnos'
             },
             AVOIN: {
-                FI: 'Avoin',
-                SV: 'Avoin'
+                FI: 'Lähetetty',
+                SV: 'Lähetetty'
             },
             VALMISTELUSSA: {
-                FI: 'Valmistelussa',
-                SV: 'Valmistelussa'
+                FI: 'Käsittelyssä',
+                SV: 'Käsittelyssä'
```

Comment:
This is fine for now. Later we could use i18n localization strings only.

Reply:
Yes, thought about it but not got any clear solution by i18n to fix it without lots of research.